lua/nvim-tree/populate.lua

```diff
@@ -46,7 +46,7 @@ local function file_new(cwd, name)
     name = name,
     absolute_path = absolute_path,
     executable = is_exec,
-    extension = vim.fn.fnamemodify(name, ':e') or "",
+    extension = string.match(name, "%.(.*)") or "",
```

Comment:
This pattern will return the full file name for dotfiles like `.gitignore`, which is incorrect. I think the pattern needs to be changed to something like `".?[^.]+%.(.*)"`.

Reply:
Yes, you're right. I didn't think about that case. I did some tests with this pattern that you suggested and it seems to work fine. Thanks!